Resources/Private/Docs/pageview.md

```diff
@@ -0,0 +1,108 @@
+# Page View
```

Comment:
This is a great work over all. 

Two remarks in this state:

1. We should not open another documentation path. Please move the pageview documentation to Documentation and use ReST as format.
2. Please rebase to the current master

I'm not yet sure how to bring together dev-extbase-fluid with this PR. But we will see.